src/HttpClientFactory/Http/src/Logging/LoggingScopeHttpMessageHandler.cs

```diff
@@ -15,15 +15,22 @@ namespace Microsoft.Extensions.Http.Logging
     public class LoggingScopeHttpMessageHandler : DelegatingHandler
     {
         private ILogger _logger;
+        private readonly Predicate<string> _isSensitiveHeader;
 
-        public LoggingScopeHttpMessageHandler(ILogger logger)
+        public LoggingScopeHttpMessageHandler(ILogger logger, Predicate<string> isSensitiveHeader)
```

Comment:
Same comments as the other handler. We avoid breaking changes like this unless really necessary, and it's not necessary here.